easel/lib/screens/creator_hub/widgets/nfts_grid_view.dart

```diff
@@ -19,15 +20,29 @@ import 'package:shimmer_animation/shimmer_animation.dart';
 
 import '../../../main.dart';
 
-class NftGridViewItem extends StatelessWidget {
+class NftGridViewItem extends StatefulWidget {
```

Comment:
Whats the reason for converting it to stateful?

Reply:
Used for `startPublishingFlowAgain` need to get context but not handling this in viewModel itself so reverting back to stateless